alembic/versions/2025_04_30_8e0b9e03054d_add_entries_for_raredisease_in_order_.py

```diff
@@ -0,0 +1,51 @@
+"""add entries for raredisease in order type application
+
+Revision ID: 8e0b9e03054d
+Revises: 039dbdf8af01
+Create Date: 2025-04-30 09:56:53.670128
+
+"""
+
+from typing import List
+
+import sqlalchemy as sa
+from sqlalchemy.orm import Session, declarative_base
+
+from alembic import op
+
+Base = declarative_base()
+
+# revision identifiers, used by Alembic.
+revision = "8e0b9e03054d"
+down_revision = "039dbdf8af01"
+branch_labels = None
+depends_on = None
+
+bind: sa.Connection = op.get_bind()
+
+
+class OrderTypeApplication(Base):
+    __table__ = sa.Table("order_type_application", sa.MetaData(), autoload_with=bind)
+
+
+def upgrade():
+    session: Session = Session(bind=bind)
+    mip_dna_rows: List[OrderTypeApplication] = (
```

Comment:
regarding my comment above
```suggestion
    mip_dna_rows: list[OrderTypeApplication] = (
```

Reply:
Check ✔️ I'll see if I can get vs code to default to that as well.